# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../qjackctl/header-fix.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -Nur qjackctl-0.2.23-orig/src/qjackctlJackConnect.h qjackctl-0.2.23/src/qjackctlJackConnect.h
--- qjackctl-0.2.23-orig/src/qjackctlJackConnect.h	2008-03-14 10:21:44.000000000 +0100
+++ qjackctl-0.2.23/src/qjackctlJackConnect.h	2008-03-14 10:23:51.000000000 +0100
@@ -22,6 +22,8 @@
 #ifndef __qjackctlJackConnect_h
 #define __qjackctlJackConnect_h
 
+#include <cstdlib>
+
 #include "qjackctlAbout.h"
 #include "qjackctlConnect.h"
 
